rewards/tree_utils.py

```diff
@@ -39,7 +38,7 @@ def calc_next_cycle_range(chain: str, tree_manager: TreeManager):
 
     # Claim at last synced block
     end_block = last_synced_block(chain)
-    if chain == "arbitrum":
+    if chain == Network.Arbitrum:
         end_block = end_block - 100
```

Comment:
Anotha one

Reply:
Good catch, plus we don't need this if statement anymore